lms/djangoapps/verify_student/views.py

```diff
@@ -1105,7 +1105,7 @@ def _send_confirmation_email(self, user):
 
         subject = _("Verification photos received")
         message = render_to_string('emails/photo_submission_confirmation.txt', context)
-        from_address = configuration_helpers.get_value('default_from_email', settings.DEFAULT_FROM_EMAIL)
+        from_address = configuration_helpers.get_value('email_from_address', settings.DEFAULT_FROM_EMAIL)
```

Comment:
@douglashall That would be much nicer, but it is out of the scope of this PR, that would also involve calling in the devops or admins of the whitelabels and have them replace the override values they are using. 

Since I am not in a position to coordinate that, I went for just using the same key in all the places the setting is called.